src/rest_framework_jwt/blacklist/admin.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.utils.translation import gettext_lazy as _
+
+from .models import BlacklistedToken
+
+
+@admin.register(BlacklistedToken)
+class BlacklistedTokenAdmin(admin.ModelAdmin):
+    list_display = ('token', )
```

Comment:
What about the other fields, such as `user` and `blacklisted_by`?

A temporal filter would be useful as well.